src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/executablestatementcount/InputExecutableStatementCountLambdas.java

```diff
@@ -0,0 +1,31 @@
+package com.puppycrawl.tools.checkstyle.checks.sizes.executablestatementcount;
+
+/* Config:
+ * max = 0
+ * tokens = LAMBDA
+ */
+
+import java.util.function.Consumer;
+
+public class InputExecutableStatementCountLambdas {
+
+    Consumer a = (o) -> { // violation
+        o.toString(); // 1
+        o.toString(); // 2
+        o.toString(); // 3
+        o.toString(); // 4
+        o.toString(); // 5
+        o.toString(); // 6
+    };
+
+    Consumer b = (x) -> { // violation
+        Consumer c = (s) -> { // violation
+            String str = s.toString();
+            str = str + "!";
+        };
+        Consumer d = (s) -> { // violation
+            String str = s.toString();
+            str = str + "?";
+        };
+    };
```

Comment:
Please also add cases like 
`a -> a `
`a -> null`

Reply:
done

Reply:
I dont see changes

Reply:
line 28

Reply:
line 28 is `Consumer t = a -> a.toString().trim()`, but I asked for `a -> a` and `a -> null`

Reply:
Oh, I thought single line lambdas were needed. Done now.